devstack/lib/nfp

```diff
@@ -366,6 +354,7 @@ function launch_configuratorVM {
     GROUP="svc_management_ptg"
     echo "GroupName: $GROUP"
     PortId=$(gbp policy-target-create --policy-target-group $GROUP $InstanceName | grep port_id  | awk '{print $4}')
+    neutron floatingip-associate $fip_id $PortId
```

Comment:
Why is floating IP associated in case of configurator VM? It should be only for visibility.